scenes/gui/tire_data_app/tire_data_circle.gd

```diff
@@ -0,0 +1,73 @@
+extends Control
+
+var tire:RaycastSuspension
```

Comment:
I have usually left a space after the colon when specifying type for variable. Could change these to match. And also move this below enums and export variables.